arkane/common.py

```diff
@@ -142,13 +149,34 @@ def __repr__(self):
         result += '}'
         return result
 
+    def as_dict(self):
+        dictionary = super(ArkaneSpecies, self).as_dict()
+        new_dictionary = {'Identifiers': {}, 'Properties': {}}
+
+        # Add additional formatting for identifiers, and properties
+        for attr, value in dictionary.items():
```

Comment:
My only concern is that we might want a different arrangement in the future, adding or changing categories, which will break compatibility. I'm OK with leaving it as is if you don't think it's an issue

Reply:
You are correct that if we decide that we want to change the name of (or get rid of for that matter) the Identifiers or Properties block, then we will have to either leave this code in or break backwards compatibility.

On the other hand, if we want to add/remove items from these blocks then we can do so without breaking backwards compatibility. This is because the code for reading in YAML files relies on the `expand_formatted_blocks` function, which takes in whatever is underneath these blocks (it does not need to know which items are allowed to be underneath the identifiers block).

I personally like having all of the Identifiers and Properties grouped together, as otherwise they can be difficult to find. However, I don't want to cause us to have to keep legacy code if we want to change this later.

I'll let you @alongd (as the principle designer of the ArkaneSpecies YAML format) decide if you think you will want to keep this change going forward. This would be simple for me to remove from this PR, and it can always be added into another PR later. Let me know what you decide, I am happy with either decision. Sorry for all of the back and forth